blog/2025/devnet-reset.md

```diff
@@ -0,0 +1,57 @@
+---
+category: 2025
+date: 2025-01-24
+seo:
+    title: Upcoming Devnet Reset Notification
+    description: Devnet is scheduled to reset on Monday, February 3, 2025 due to Clio databases nearing capacity. Learn more.
+labels:
+    - Advisories
+markdown:
+    editPage:
+        hide: true
+---
+# Upcoming Devnet Reset
+
+Devnet is scheduled for a reset on **Monday, February 3, 2025**. Clio server databases on the network are nearing storage capacity and need to be cleared.
+
+<!--
+{% admonition type="success" name="Update" %}
+The reset has completed successfully. Devnet is online and fully operational.
+{% /admonition %}
+-->
+
+## Background
+
+Ripple operates Clio servers on Devnet to help the community access validated ledger data efficiently. It better handles API calls and stores data in a more space-efficient format than rippled servers, ultimately reducing the load on the network. Clio uses ScyllaDB, which is nearing storage capacity; in order to continue serving the community, Devnet will be reset to clear the Clio databases.
+
+## Impact
+
+This reset affects Devnet only. Other networks will continue to operate as usual, including XRPL Mainnet, XRPL Testnet, Xahau, and the Hooks Testnet.
+
+The reset will delete all ledger data in Devnet, including all accounts, transactions, balances, settings, offers, AMMs, escrows, and other data. This means all balances will be reset to zero and the block number will start at one again. No changes are anticipated to services such as Devnet APIs, faucets, Explorers, access rights, and wallet integrations; these services usually manage resets without issues.
+
+Devnet will have its amendment statuses restored to match XRPL Mainnet. Any existing accounts or other data will need new test XRP from the faucet and will need to be re-created. 
+
+If code relies on specific addresses, a request to the faucet can fund the same address again. However, any AMMs that are re-created after the reset will generally have different account addresses. As a reminder, it's best not to use the same addresses or key pairs on Mainnet and any developer networks.
+
+The important URLs, Network ID, and validator settings for Devnet will remain the same.
```

Comment:
I think the validator settings for Devnet are expected to change with the new validator list signing key (if we do that).

Reply:
They're upading t he validator keys--adding instructions for that.